apps/icelandic-names/backend/bin/startup.sql

```diff
@@ -0,0 +1,15 @@
+\set db_user `echo $DEV_DB_USER`
+\set db_password `echo $DEV_DB_PASS`
+\set db_name `echo $DEV_DB_NAME`
+\set test_db_user `echo $TEST_DB_USER`
+\set test_db_password `echo $TEST_DB_PASS`
+\set test_db_name `echo $TEST_DB_NAME`
```

Comment:
Should test config be removed here and added back when adding relevant tests to ensure the has no unused config/code in case the test written don't depend on this config?
(applies to all following test config/code)